src/pmsearch/pmsearch_daily.sh

```
@@ -0,0 +1,234 @@
+    SRCFLAGS="-a $ARCHIVE"
+else
+    SRCFLAGS=""
+fi
+
+if $VERBOSE
+then
+    echo "$prog: index target: $INDEX"
+fi
+
+# Build metric→indom mapping first (needed for both metrics and instances).
+pminfo $SRCFLAGS -I 2>/dev/null | $PCP_AWK_PROG '
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)